src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctedRobot = {};
+  const robotValues = Object.values(robot);
+
+  // Check if values are not repeated
+  for (const value of robotValues) {
+    if (robotValues.indexOf(value) !== robotValues.lastIndexOf(value)) {
+      return null;
+    }
+  }
+
+  // Change keys and values on places
+  for (const key in robot) {
+    const robotValue = robot[key];
+
```

Comment:
and add check if !correctedRobot[robotValue]

Reply:
Added